kong/plugins/prometheus/schema.lua

```diff
@@ -14,6 +14,7 @@ return {
         type = "record",
         fields = {
           { per_consumer = { type = "boolean", default = false }, },
+          { kong_http_status_with_target = { type = "boolean", default = false }, },
```

Comment:
This breaks version compatibility with older data planes; updates will need to be made to the [compatibility layer ](https://github.com/Kong/kong/blob/master/kong/clustering/compat/removed_fields.lua)to handle these changes.

@aboudreault another version compatibility issues to watch out for.